cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -1027,38 +1029,41 @@ def _validate_obsm(self):
             if " " in key:
                 self.errors.append(f"Embedding key {key} has whitespace in it, please remove it.")
```

Comment:
Is this a requirement for embeddings that do not start with "X_"?

Reply:
My understanding from the discussion on the ticket was that whitespace should never be allowed in any embedding key, but that definitely isn't clear from the schema definition. I'll make sure to bring it up with Signe/Brian/Jason but I am reasonably confident from an implementation perspective we don't want to allow whitespace characters in any embedding key.

Reply:
As written, it's only for X_ - cases that **might** be presented in Explorer.  I don't see why we would enforce naming conventions on embeddings intended for use in other scenarios.

Reply:
CC: @jahilton 

Also - the warnings were an implementation detail rather than a schema requirement per the top-level note:

**Note: @jahilton has requested that obsm embeddings that do not start with X_ also be validated against the CXG requirements in X_{suffix} and generate warnings that include the embedding name. This is not a schema requirement because additional embeddings may be present for [other scenarios](https://github.com/chanzuckerberg/single-cell-curation/issues/590#issuecomment-1735947750), but an implementation requirement to provide information to the curators to assess the state of embeddings in the dataset.  @joyceyan has noted this in #654.**